flask/run.sh

```diff
@@ -29,4 +36,8 @@ fi
 
 gcloud compute ssh redis-relay --zone=us-central1-a -- -N -L $FLASK_LOCAL_REDISPORT:$FLASK_REDISHOST:6379 &
 
+sleep 1 
+
+celery -A src.queues.email_subscribe worker -l INFO &
```

Comment:
Also, don't wee need to kill this one too?

IIRC when you use & to background a process it continue running even if the parent shell script terminates.